src/main/scala/com/stratio/hermes/utils/Hermes.scala

```diff
@@ -16,55 +16,155 @@
 
 package com.stratio.hermes.utils
 
+import java.security.InvalidParameterException
+
 import com.stratio.hermes.constants.HermesConstants
-import com.stratio.hermes.helpers.RandomHelper
 import com.stratio.hermes.implicits.HermesSerializer
 import com.stratio.hermes.models.NameModel
 import org.json4s._
 import org.json4s.native.Serialization.read
-
-import scala.util.Try
+import com.stratio.hermes.helpers.RandomHelper
+import scala.language.postfixOps
+import scala.util.{Random, Try}
 
 /**
- * Hermes util used for to generate random values.
- */
+  * Hermes util used for to generate random values.
```

Comment:
I think we should all set-up our IDEs the same way to avoid these changes in code that we haven't modified.